mami/script.js

```diff
@@ -0,0 +1,25 @@
+let userName = 'Alex'
+let userEmail = 'a.gmail.com'
+let userPassword = 'Alex22'
+let userConfirmPassword = 'alex22'
+
+
+// * Check if the userName length bigger than 3 chr.
+console.log(userName.length>3)
+
+// * Check if the userEmail length bigger than 12 chr and includes @.
```

Comment:
You forgot to solve 3rd Task! 
Check if the userEmail length bigger than 12 chr and includes @.